internal/cmd/schema-registry/command_cluster_describe.go

```diff
@@ -178,5 +192,5 @@ func schemaCountQueryFor(schemaRegistryId string) metricsv2.QueryRequest {
 }
 
 func getMaxSchemaLimitPriceKey(sgPackage, serviceProvider, serviceProviderRegion string) string {
```

Comment:
Nit: Pass arguments in the same order they're used.
```suggestion
func getMaxSchemaLimitPriceKey(serviceProvider, serviceProviderRegion, sgPackage string) string {
```